Installation-SUSE-CAP-1.0.md

```diff
@@ -104,7 +104,7 @@ helm repo add suse https://kubernetes-charts.suse.com/
 ## Deploy Stratos, the Console UI
 
 ```
-helm install suse/console --namespace=stratos-cap --values scf-config-values.yaml
+helm install suse/console --namespace=stratos --values scf-config-values.yaml
```

Comment:
You changed the stratros namespace here but didn't adapt it in the SES preparation part of the `Configuring the deployment` section. This needs to be in sync.

Reply:
Yep. Cool, thanks - LGTM